storage-shared-benchmarking/src/main/java/com/google/cloud/storage/benchmarking/StorageSharedBenchmarkingCli.java

```diff
@@ -108,35 +120,68 @@ private void runWorkload1() {
     StorageOptions retryStorageOptions =
         StorageOptions.newBuilder().setProjectId(project).setRetrySettings(retrySettings).build();
     Storage storageClient = retryStorageOptions.getService();
-    runW1R3(storageClient);
+    try {
+      ListeningExecutorService executorService =
```

Comment:
IIUC, this code block was factored out of runW1R3; and to only sample once but created a copy for workload4 as well; Can you factor in sampling as an option to W1R3() and only print a sample if not a warm up instead?

Reply:
can you rephrase this ask?

Reply:
refactor runW1R3() to accept "isSample" (bool) and "samples" (int) to reduce duplication